parts/linux/cloud-init/artifacts/mig-partition.sh

```diff
@@ -21,7 +21,7 @@ case ${MIG_PROFILE} in
         ;;  
     *)
         echo "not a valid GPU instance profile"
-        exit ${ERR_MIG_PARTITION_FAILURE}
```

Comment:
this variable is invalid in this context, it's sourced from CSE and this is a systemd unit. switch to exit 1.